packages/preview/mass-spec/0.1.0/LICENSE

```diff
@@ -0,0 +1,20 @@
+MIT License
+Copyright (c) 2023 Satvik Saha, DVDTSB
```

Comment:
Are the names in the copyright correct or this a copy-paste error?

Reply:
Not at all! I'll update :)